rotkehlchen/accounting/transactions.py

```diff
@@ -143,7 +147,7 @@ def _process_tx_swap(
             given_price=prices[0],
             count_entire_amount_spend=False,
             extra_data={
-                'tx_hash': out_event.tx_hash.hex(),
+                'tx_hash': base_group_id,  # TODO: undo. Is a temporary hack
```

Comment:
It is

Reply:
I mean the key on the `extra_data` attribute

Reply:
What I did right now is I made the functionality the way it was at the moment of rotki 1.27.1 version. Not sure how event identifier will be useful for the frontend and for the user. Though I get that putting event identifier in tx_hash field is also strange, but this is the way it was before and since this PR is just a temporary fix, I would say that it will be better to stay with what we had before.